aten/src/ATen/native/cuda/MultiMarginLoss.cu

```diff
@@ -148,7 +148,7 @@ void multi_margin_loss_shape_check(
 
     TORCH_CHECK(
         target.dim() <= 1 && target.numel() == nframe,
-        "inconsistent target size, expected ", nframe, " but got ",
+        "MultiMarginLoss: The size of input tensor ", nframe, " must match the size of target tensor ",
         target.sizes());
```

Comment:
nit: I think this might be slightly clearer
```suggestion
        "MultiMarginLoss: The  batch size of input tensor, ", nframe, ", must match the size of target tensor, but got target of size",
        target.sizes());
```